content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -256,6 +272,15 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
 
    If you do not see an error, it means the plugin is successfully installed.
 
+   {{< note >}}
+   The curl commands processed in the previous steps will remain on your target directory indefinitely and will not be utilized after the kubectl install (step 3) is complete.
+
+   ```bash
+   rm kubectl-convert kubectl-convert.sha256
+   ```
+
+   {{< /note >}}
```

Comment:
Instead of adding these as notes, make it the next numbered step, worded like 

After installing the plugin, clean up the installation files:

\<command\>

Reply:
Resloved this